.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -1069,3 +1069,20 @@ and non interactive tsh bench loads.
 - Verify tsh runs on:
   - [ ] Windows 10
   - [ ] MacOS
+
+## Host users creation
+
+[Host users creation docs](https://github.com/gravitational/teleport/pull/13056)
```

Comment:
Can we get the docs PR merged first and then add a link to the actual docs? Or leave a TODO with the proper link?
```suggestion
[Host users creation docs](../../docs/pages/server-access/guides/host-user-creation.mdx)
```